crates/stress-test/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "stress-test"
+edition.workspace = true
+version.workspace = true
+license.workspace = true
+readme.workspace = true
+repository.workspace = true
+homepage.workspace = true
+
+[dependencies]
+anyhow = "1"
+trapeze = "0.7"
+prost = "0.13"
+prost-types = "0.13"
+tokio = { workspace = true, features = ["macros", "fs", "process", "signal"] }
+humantime = "2.1.0"
+tempfile = "3"
+oci-spec.workspace = true
+containerd-shim-wasm-test-modules.workspace = true
+clap = { version = "4", features = [ "derive"] }
+log.workspace = true
```

Comment:
I think we should consider making a consistent format. The one we've been using in the shims are

```toml
anyhow = { workspace = true }
```